### DISCUSSION TITLE
Report 18+ Bots for Porn instead of Spam

### DISCUSSION THREAD

**Discussion by Lightning11wins** [ThioJoe/YT-Spammer-Purge#1052] [Ideas]
How hard would it be to make some of the spam filters for 18+ and other porn bots report them for that instead of/in addition to spam? Maybe we could also throw in reporting the channel, profile picture, and channel art, since most 18+ bots violate YT guidelines on them, too.

Hopefully, that would help YouTube to train its filters more effectively. Would it be difficult to implement?

**Comment by conorcoder556** [comment]
I really totally agree╰(*°▽°*)╯
